exercises/saddle-points/instructions.md

```diff
@@ -5,7 +5,7 @@ Your task is to find the potential trees where you could build your tree house.
 The data company provides the data as grids that show the heights of the trees.
 The rows of the grid represent the east-west direction, and the columns represent the north-south direction.
 
-An acceptable tree will be the largest in its row, while being the smallest in its column.
+An acceptable tree will have the greatest height in its row, while having the least height in its column.
```

Comment:
```suggestion
An acceptable tree will the tallestt in its row, while being the shortest  n its column.
```

Reply:
@kotp "will the tallest" should that be "will be the tallest"?

Reply:
@MatthijsBlom What about the above suggestion?

Reply:
Couldn't we use the phrasing from the introduction?

Reply:
We could. I figured the difference between `introduction.md` and `instructions.md` might be deliberate, so I kept them different.

Reply:
I'm fine with unifying them, especially when it turns out that the terminology is so hard to get right.

Reply:
If we use the same wording in both cases, we could also de-duplicate.

One potential advantage of using two different descriptions: these provide different perspectives on the same problem.

Reply:
I did not agree with applying this suggestion, but it was applied anyway.

The point of my change was to fix an inaccuracy. This now applied suggestion effectively reverts my change and _reintroduces_ the original inaccuracy. This inaccuracy has previously irritated people, hence this PR. See the forum thread, and the other threads here.